lib/workers/repository/model/custom-commit-message.spec.ts

```diff
@@ -0,0 +1,51 @@
+import { CustomCommitMessage } from './custom-commit-message';
+
+describe('workers/repository/model/custom-commit-message', () => {
+  describe('CustomCommitMessage', () => {
+    const TEST_CASES: ReadonlyArray<
+      [message: string, prefix: string | undefined, result: string]
+    > = [
+      ['test', '', 'Test'],
+      ['  test  ', '  ', 'Test'],
+      ['test', 'fix', 'fix: test'],
+      ['test', 'fix:', 'fix: test'],
+      [
+        'Message    With   Extra  Whitespaces   ',
+        '  refactor   ',
+        'refactor: message With Extra Whitespaces',
+      ],
+    ];
+
+    it.each(TEST_CASES)(
```

Comment:
can we please use the string template table syntax here.

Reply:
Done